src/main/java/org/spongepowered/api/data/DataTransactionResult.java

```diff
@@ -330,7 +330,7 @@ public boolean isSuccessful() {
      *
      * @param consumer The consumer to call
      */
-    public void ifSucessful(Consumer<List<ImmutableValue<?>>> consumer) {
```

Comment:
This method should be depreciated (with a `@depreciated` tag explaining why in the javadocs), rather than outright removed, and call the correct method - [a bit like what was done here](https://github.com/SpongePowered/SpongeAPI/blob/0477faf48f47f93b83d02a9c9a8370d22d685206/src/main/java/org/spongepowered/api/item/inventory/property/InventoryDimension.java#L39-L44). We'd then remove it two API versions up the road (so, if it was pulled now, we would want to deprecate for API 7, which we'd then remove in API 8)